src/patterns/ec2-app.ts

```diff
@@ -203,12 +192,24 @@ export class GuEc2App {
 
     const listener = new GuHttpsApplicationListener(scope, "Listener", {
       app,
-      open: props.access.scope === AccessScope.PUBLIC, // this is true as default, which adds an ingress rule to allow all inbound traffic
```

Comment:
This was confusing me because [`open`](https://docs.aws.amazon.com/cdk/api/latest/docs/@aws-cdk_aws-elasticloadbalancingv2.ApplicationListener.html#open) is essentially doing the same thing as manually adding a `GuPublicInternetAccessSecurityGroup`.